src/beamlime/config/raw_detectors/bifrost.py

```diff
@@ -75,6 +77,18 @@ def make_spectrum_view(data: DetectorData[SampleRun]) -> SpectrumView:
     )
 
 
+def make_counts_per_angle(
+    data: DetectorData[SampleRun], rotation: DetectorRotation
+) -> CountsPerAngle:
+    edges = sc.linspace('angle', 0, 91, num=46, unit='')
+    da = sc.DataArray(
+        sc.zeros(dims=['angle'], shape=[45], unit='counts'), coords={'angle': edges}
+    )
+    counts = sc.values(data.sum().data)
+    da['angle', rotation.data[-1]] += counts
```

Comment:
What happens if the rotation given on input is outside of the 0-91 range? Can that ever happen?